pkg/security/tests/schemas.go

```diff
@@ -88,6 +88,11 @@ func validateExecSchema(t *testing.T, event *sprobe.Event) bool {
 	return validateEventSchema(t, event, "file:///schemas/exec.schema.json")
 }
 
+//nolint:deadcode,unused
+func validateExitSchema(t *testing.T, event *sprobe.Event) bool {
+	return validateEventSchema(t, event, "file:///schemas/exit.schema.json")
```

Comment:
nit - use `t.Helper` so the stack is adjusted to show the line in the actual test.
```suggestion
	t.Helper()
	return validateEventSchema(t, event, "file:///schemas/exit.schema.json")
```

Reply:
I've added the `t.Helper()` and it didn't change the test output.

When `validateExitSchema` is called, we're setting an error to the test so we have the reference to where it was called from:
https://github.com/DataDog/datadog-agent/blob/ca114ac9b11890cb8257a861a3fd51f8ec591c36/pkg/security/tests/process_test.go#L1239-L1241

Since the other `validate*Schema` functions are not using `t.Helper()`, I'll keep `validateExitSchema` without it for now. What do you think?

Reply:
I made https://github.com/DataDog/datadog-agent/pull/12887 to help with this

Reply:
So yes let's keep it without the t.Helper for now